scipy/sparse/_sputils.py

```diff
@@ -1,7 +1,9 @@
 """ Utility functions for sparse matrix module
 """
 
+from collections.abc import Iterable
 import sys
+from typing import Any, Literal as L, Optional, Union
```

Comment:
Let's avoid aliasing `Literal` for now, unless this is common throughout the rest of the codebase.

Reply:
Done!